src/components/theme/Widgets/ArrayWidget.jsx

```diff
@@ -5,14 +5,14 @@ const ArrayWidget = ({ value, children, className }) =>
   value ? (
     <span className={cx(className, 'array', 'widget')}>
       {value.map((item, key) => (
-        <>
+        <React.Fragment key={item.token || item.title || item}>
```

Comment:
I think we should just use the array index here. There can be multiple items in an array with the same value.